general/releases/2.4/2.4.2.md

```diff
@@ -0,0 +1,66 @@
+---
+title: Moodle 2.4.2
+tags:
+  - Release notes
+  - Moodle 2.4
+sidebar_position: 2
+moodleVersion: 2.4.2
+---
+<p class="note">Because of a serious regression in the 2.4.2 release, [Moodle 2.4.3](/general/releases/2.4/2.4.3) was release on Mon 18 March, 2013.</p>
```

Comment:
```suggestion
:::note

Because of a serious regression in the 2.4.2 release, [Moodle 2.4.3](/general/releases/2.4/2.4.3) was released on Mon 18 March, 2013.

:::
```